pyproject.toml

```diff
@@ -1,10 +1,257 @@
 [build-system]
 requires = [
-    "setuptools",
-    "wheel",
+  "hatchling>=1.8.0",
 ]
-build-backend = "setuptools.build_meta"
+build-backend = "hatchling.build"
 
+[project]
+name = "farm-haystack"
+dynamic = [
+  "version",
+]
+description = "Neural Question Answering & Semantic Search at Scale. Use modern transformer based models like BERT to find answers in large document collections"
+readme = "README.md"
+license = "Apache-2.0"
+requires-python = ">=3.7, <4"
```

Comment:
Ok to remove the capping here, we have a several more around but some are needed, I'll address those later

Reply:
Awesome, removed!